lib/ansible/modules/cloud/docker/docker_network.py

```diff
@@ -308,6 +377,7 @@ def create_network(self):
                                                   driver=self.parameters.driver,
                                                   options=self.parameters.driver_options,
                                                   ipam=ipam_config,
+                                                  enable_ipv6=self.parameters.enable_ipv6,
```

Comment:
Could you convert the arguments of `create_network()` into a dict `params`, and pass it using the `**params` syntax? That makes it simpler to only add parameters if they are really needed (like `enable_ipv6`; only pass that if it is not `None`).

Reply:
Done